core/src/main/java/org/apache/calcite/util/format/FormatElementEnum.java

```diff
@@ -221,6 +245,33 @@ public enum FormatElementEnum implements FormatElement {
       sb.append(String.format(Locale.ROOT, "%02d", calendar.get(Calendar.WEEK_OF_YEAR)));
     }
   },
+  V("", "The number of the year (Monday as the first day of the week),"
```

Comment:
the comment seems wrong, it is about the week number

Reply:
+1， i will fix it